File: Geshotel/Geshotel.Web/Modules/Common/Imports/ServerTypings/Contratos.EdadesForm.ts
namespace Geshotel.Contratos {
    export class EdadesForm extends Serenity.PrefixedContext {
        static formKey = 'Contratos.Edades';

    }

    export interface EdadesForm {
        HotelId: Serenity.LookupEditor;
        ClienteId: Serenity.LookupEditor;
        FechaDesde: Serenity.DateEditor;
        FechaHasta: Serenity.DateEditor;
        TipoHuespedId: Serenity.LookupEditor;
        EdadMinima: Serenity.IntegerEditor;
        EdadMaxima: Serenity.IntegerEditor;
    }

    [['HotelId', () => Serenity.LookupEditor], ['ClienteId', () => Serenity.LookupEditor], ['FechaDesde', () => Serenity.DateEditor], ['FechaHasta', () => Serenity.DateEditor], ['TipoHuespedId', () => Serenity.LookupEditor], ['EdadMinima', () => Serenity.IntegerEditor], ['EdadMaxima', () => Serenity.IntegerEditor]].forEach(x => Object.defineProperty(EdadesForm.prototype, <string>x[0], { get: function () { return this.w(x[0], (x[1] as any)()); }, enumerable: true, configurable: true }));
}

File: Geshotel/Geshotel.Web/Modules/Contratos/ContratosEdades/ContratosEdadesForm.cs
﻿
namespace Geshotel.Contratos.Forms
{
    using Serenity;
    using Serenity.ComponentModel;
    using Serenity.Data;
    using System;
    using System.ComponentModel;
    using System.Collections.Generic;
    using System.IO;

    [FormScript("Contratos.ContratosEdades")]
    [BasedOnRow(typeof(Entities.ContratosEdadesRow))]
    public class ContratosEdadesForm
    {
        public Int16 HotelId { get; set; }
        public Int32 ClienteId { get; set; }
        public DateTime FechaDesde { get; set; }
        public DateTime FechaHasta { get; set; }
        public Int16 TipoHuespedId { get; set; }
        public Int32 EdadMinima { get; set; }
        public Int32 EdadMaxima { get; set; }
        //public Int32 UserId { get; set; }
        //public DateTime FechaModificacion { get; set; }
    }
}